src/cdk/overlay/overlay-directives.ts

```diff
@@ -326,7 +326,8 @@ export class CdkConnectedOverlay implements OnDestroy, OnChanges {
       overlayX: currentPosition.overlayX,
       overlayY: currentPosition.overlayY,
       offsetX: currentPosition.offsetX || this.offsetX,
-      offsetY: currentPosition.offsetY || this.offsetY
+      offsetY: currentPosition.offsetY || this.offsetY,
+      panelClass: currentPosition.panelClass || undefined,
```

Comment:
[The panel class is already being passed in here](https://github.com/angular/material2/blob/master/src/cdk/overlay/overlay-directives.ts#L315). Putting it on the position itself shouldn't be necessary.

Reply:
> Looks like the PR is labelled for `dialog`, even though it doesn't have anything to do with the dialog.
Oh sorry, made a mistake - supposed to be overlay. Should I create a new PR or changing the title is okay?




> [The panel class is already being passed in here](https://github.com/angular/material2/blob/master/src/cdk/overlay/overlay-directives.ts#L315). Putting it on the position itself shouldn't be necessary.

That is coming from the input, positions can have their own panel class (and weight, which is not also passed down) - https://github.com/angular/material2/blob/master/src/cdk/overlay/position/flexible-connected-position-strategy.ts#L1179

Reply:
Ah, sorry. I just re-read it and it does make sense.